src/pages/Register.ts

```diff
@@ -246,7 +246,7 @@ const Register = () => {
         busy &&
           centeredSpinner({
             size: 34,
-            ...{ style: { display: null, margin: null, marginLeft: '1ex' } },
+            ...{ style: { display: undefined, margin: undefined, marginLeft: '1ex' } },
```

Comment:
Could our SpinnerOptions override the display and margin props so that these don't need to explicitly undefined?

Reply:
SpinnerOptions does allow for passing a `style` object. These `undefined`s are unsetting the values set by `centeredSpinner`. I suspect that `spinner` might fit better here than `centeredSpinner`.